src/components/composer/composer/composer.ts

```diff
@@ -572,3 +572,12 @@ export class Composer extends Component<Props, SpreadsheetChildEnv> {
     this.processTokenAtCursor();
   }
 }
+
+Composer.props = {
+  inputStyle: { type: String, optional: true },
+  rect: { type: Object, optional: true },
+  delimitation: { type: Object, optional: true },
+  focus: { type: String, optional: true },
```

Comment:
To be really thorough, we should go as far as this:
```suggestion
 focus: { validate:  v => ["inactive", "cellFocus", "contentFocus"].includes(v), optional: true },
```
  or maybe
``` typescript  
const focusValues = ["inactive", "cellFocus", "contentFocus"] as const;
  ...
interface Props {
...
focus: typeof  focusValues[number]
}

Compser.props = {
    ...,
    focus: { validate:  v => focusValues.includes(v), optional: true },
}
```
Redundant here in TS but addresses the actual validation ^^  That being said, we have to stop somewhere to avoid typing every object passed as props. it's just that this case is kind of specific and easy to implement